packages/web/src/builders/package/run-rollup.ts

```diff
@@ -1,10 +1,15 @@
 import * as rollup from 'rollup';
-import { from, of } from 'rxjs';
-import { catchError, map, switchMap } from 'rxjs/operators';
+import { from } from 'rxjs';
+import { map, concatMap } from 'rxjs/operators';
 
 export function runRollup(options: rollup.RollupOptions) {
   return from(rollup.rollup(options)).pipe(
-    switchMap(bundle => from(bundle.write(options.output))),
+    concatMap(bundle => {
+      const outputOptions = Array.isArray(options.output)
```

Comment:
`output` option was changed in rollup to support arrays, so we just map all of them to `bundle.write` promises and await them all.

Reply:
was this a breaking change in rollup? or is it just more efficient than just handing all the outputs to rollup like we did previously?

Reply:
Wasn't a breaking change per se, but rollup now allows you to pass an array of output format. Now that I think about it, I should refactor the code to take advantage of that. :)